stable/graylog/README.md

```diff
@@ -218,13 +226,10 @@ graylog:
       -----END PRIVATE KEY-----
 ```
 
-Then configure Graylog input to
+By default this will configure Graylog to run in TLS mode and mount the certificates in the `/etc/graylog/server` directory.
```

Comment:
To avoid the confusion, the Web UI TLS should be put on another section. This section should contains only Input TLS detail.